bool DAQ_on
---
bool was_set


